### ISSUE TITLE
2022-12-01 Youtube has Blocked XDM when logged into utube account

### ISSUE THREAD

**Issue by i-hate-utube** [subhra74/xdm#939]
XDM2020
Latest Firefox 106.0.5 64 Bit
linux Mint mate 19.3 64BIT
XDM integration module for Firefox Quantum v2.2
Version 7.2.11 with Java Eclipse OpenJ9 11.0.6 on Linux

**Describe the bug**
When logged into utube account and after loading up the link and select video in download box u get this error 

Download Failed
"XDM could not download the file as server sent an invalid response"

*Expected behavior**
I expected the download to happen DOHHHHHHH.

**Comment by Rolf-K** [comment]
Can confirm that. (Using XDM logged out from YT in a private window while logged in a normal works)

**Comment by i-hate-utube** [comment]
I have used xdm for over 2 years and I always used it while logged in.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------

**Comment by i-hate-utube** [comment]
Private Windows is now doing it as of the 2nd Dec 22, we need an update! screw google!

**Comment by smestarz** [comment]
I use Firefox 107.0.1 on Windows 10 pro 64 Bit and when tried to download the youtube video it gives me an error
"XDM could not download the file as server sent invalid response" this started on 3rd Dec 2022. can you please check?
So this issue is happening as youtube has probably changed something within.

**Comment by SlasherII** [comment]
Confirm here also. Youtube does not work with XDM on either my Windows 10 or Linux Fedora machines. Bitchute and Rumble for example does work to download from. 
Error is "XDM could not download the file as server sent invalid response"
Checked this morning that I used the latest version of XDM.

**Comment by Rolf-K** [comment]
Seems broken when not logged in as well now.

**Comment by inuonly** [comment]
> Seems broken when not logged in as well now.

![image](https://user-images.githubusercontent.com/26686197/206399215-509fd526-16e8-41dc-a542-9e7bd3f3f5cf.png)
yes , its a complete blackout from utube for xdm is what currently looks like. No downloads not even in private window.

**Comment by tt-thoma** [comment]
Youtube has been making a lot of changes recently, including "patching" the latest version of Vanced (making it unusable) and is promoting a lot Youtube Premium (I won't be surprised if they make dislike count only available for Premium users).
I think they purposely block the library used in XDM to download Youtube videos so they can get more money (they like money) from their Premium subscription, where you can download videos.

**Comment by Damilarondo** [comment]
Yeah YouTube seemed to have had an update, you may need to download the experimental versions of XDM it works just fine on the experimental versions 
Link: https://github.com/subhra74/xdm-experimental-binaries/tags
